packages/stack-server/src/app/(main)/(protected)/projects/[projectId]/users/page-client.tsx

```diff
@@ -1,24 +1,98 @@
 "use client";
-
+import React from "react";
+import * as yup from "yup";
 import { useAdminApp } from "../use-admin-app";
 import { PageLayout } from "../page-layout";
 import { Alert } from "@/components/ui/alert";
 import { StyledLink } from "@/components/link";
 import { UserTable } from "@/components/data-table/user-table";
+import { Button } from "@/components/ui/button"
+import { SmartFormDialog } from "@/components/form-dialog";
+import { ActionDialog } from "@/components/action-dialog";
+import Typography from "@/components/ui/typography";
+
 
+type CreateDialogProps = {
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+  opennotification: Function
```

Comment:
how about this 
  setShowNotifyPasswordDialog: React.Dispatch<React.SetStateAction<boolean>>;
  instead of 
   onOpenChange: (open: boolean) => void;
   will this work